debezium-connector-mariadb/src/test/java/io/debezium/connector/mariadb/DefaultValueTest.java

```diff
@@ -25,7 +25,7 @@
 public class DefaultValueTest extends BinlogDefaultValueTest<MariaDbValueConverters, MariaDbAntlrDdlParser> {
     @Override
     protected MariaDbAntlrDdlParser getDdlParser(MariaDbValueConverters valueConverters) {
```

Comment:
```suggestion
    protected MariaDbAntlrDdlParser getDdlParser() {
```
This and any other references to `getDdlParser()` function in `BinlogDefaultValueTest` imo.